pages/features.html

```diff
@@ -6,104 +6,134 @@
+      {% trans "Set hotcues to mark places in tracks. Remix and mash up tracks with accurate, rapid fire hotcue triggering." %}
     </p>
   </div>
-
   <div class="halfbox_right">
     <img width="362px" src="{% static '/static/images/feature_waveform-2.0.png' %}">
     <p><b>{% trans "Beat Rolls and Censor" %}</b><br>
-      {% trans "Play with rhythm by triggering short loops and reverse-playback effects.  The track stays in time so you don't miss a beat." %}
+      {% trans "Play with rhythm by triggering short loops and reverse-playback effects. The track stays in time so you don't miss a beat." %}<!--TODO: Write a better way of explaining slip mode.-->
     </p>
 
-    <p><b>{% trans "Intuitive Pitchbend" %}</b><br>
-      {% trans "Ramping pitchbend ensures any tempo nudging is smoothly applied, just like with vinyl." %}
+    <p><b>{% trans "Quantization" %}</b><br>
+      {% trans "Mark and activate cue points and loops exactly on beat every time." %}
     </p>
 
     <p><b>{% trans "Broad Format Support" %}</b><br>
-      {% trans "Our mature audio decoders deliver pristine sound with rock-solid stability. Mixxx plays MP3, M4A/AAC, Ogg Vorbis, Opus, FLAC, WAVE, and AIFF formats." %}
+      {% trans "Compatibile with music files in lossless FLAC, WAV, and AIFF formats as well as lossy MP3, M4A/AAC, Ogg Vorbis, and Opus formats." %}
     </p>
 
     <p><b>{% trans "EQ and Crossfader Control" %}</b><br>
-      {% trans "Mixxx features multiple choices of audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades" %}.
+      {% trans "Choose between multiple audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades." %}
     </p>
   </div>
-
-  <div style="clear: both;"></div>
-
-  <div>
-    <H2>{% trans "Quad Sampler Decks" %}</H2>
+  
+    <div style="clear: both;"></div>
+      <h2>{% trans "Quad Sampler Decks" %}</h2>
     <p>
       {% trans "Perfect for dropping that vocal sample or airhorn, our quad sampler decks are routable through either playback channel." %}
     </p>
 
     <center><img src="{% static '/static/images/feature_sampler2.png' %}"></center>
-
-    <H2>{% trans "Built-in Effects" %}</H2>
+    
+    <h2>{% trans "Built-in Effects" %}</h2>
     <p>
-      {% trans "Add sizzle to your mix with reverb, echo, flange, and more.  Assign a Quick Effect to your controller's filter knob to add color to your sound instantly.  Bad PA system or lousy room?  Use the built-in master EQ to smooth out the sound." %}
+      {% trans "Use reverb, echo, flange, and more effects to blend music in your own signature ways. Assign a Quick Effect to your controller's filter knob to add color to your sound instantly.  Bad PA system or lousy room?  Use the built-in master EQ to smooth out the sound." %} <!--TODO: Say a little more-->
```

Comment:
Speaking of effects, could you update the VCI-400 mapping for the new effects UI? There is some [documentation](http://mixxx.org/wiki/doku.php/updating_controller_mappings) for updating mappings on the wiki.